spec/javascripts/Settings/components/Common/__snapshots__/TextInputGroup.spec.jsx.snap

```diff
@@ -1,24 +1,69 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`should render correctly 1`] = `
-<FormGroup
-  fieldId="service_proxy_attributes_dooz_kawa_chanson_input"
-  helperText="Enter your favourite Dooz Kawa tune"
+<TextInputGroup
```

Comment:
I know this doesn't quite belong to this PR **BUT** the `package-lock` migration to `yarn.lock` migration resulted to be quite not straightforward with the dependencies versions. It ended upgrading, besides many other packages, Patternfly **minor** version... which it introduces the use of references that React suggest to libraries maintainers to release it as a **major** version upgrade. SO, live with it.

Reply:
Did you use `yarn import` to generate the yarn.lock?

https://classic.yarnpkg.com/blog/2018/06/04/yarn-import-package-lock/

Reply:
Yes, I did

Reply:
Why not coping with the upgrade in another PR? Either install the older version by hard here or upgrade to the latest in master, with the change, and rebase this one.

Reply:
I tried to do it, but it triggered some incompatibility issues with other packages. In the end installing many versions of other dependencies... Was a waste. I think we can totally cope with this.

Reply:
Yeah well, it's not a big deal